addon/components/draggable-object.js

```diff
@@ -2,7 +2,6 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   dragCoordinator: Ember.inject.service(),
-  tagName: "div",
```

Comment:
Components are `div` by default, so there's no need to declare it here. Let me know if there was reasoning behind this.